sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkDoubleCounter.java

```diff
@@ -19,8 +20,12 @@
 import io.opentelemetry.sdk.metrics.internal.state.MeterSharedState;
 import io.opentelemetry.sdk.metrics.internal.state.WriteableMetricStorage;
 import java.util.function.Consumer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 final class SdkDoubleCounter extends AbstractInstrument implements DoubleCounter {
+  private static final ThrottlingLogger logger =
```

Comment:
I guess it's better to have a static Logger, non-static ThrottlingLogger to prevent logs from instrument blocking another one forever based on the timing.